README.md

```diff
@@ -49,16 +49,31 @@ Running the linting:
 ### Running the tests
 
 Mocha is used for writing tests.
+
 Run them with:
+
 ```bash
 $ yarn test
 ```
 
 For functional testing:
+
 ```bash
 $ yarn test:routes
 ```
 
+For accessibility testing:
+
+```bash
+$ yarn test:a11y
+```
+
+For test coverage:
+
+```bash
+$ yarn test:coverage
+```
```

Comment:
probably worth adding this section for now to the README:
https://github.com/hmcts/cmc-legal-rep-frontend#warnings-while-running-yarn-install-on-yarn-version-101

Reply:
Saw the warning but it does not break anything. My approach would be the same as regarding other warnings, I mean ignore ;) I do not like the fact somebody has to remove it at some point.

```
warning "request-promise-native@1.0.4" has incorrect peer dependency "request@^2.34".
warning "request-promise-core@1.1.1" has incorrect peer dependency "request@^2.34".
warning "react-dom@15.6.1" has incorrect peer dependency "react@^15.6.1".
warning "ajv-keywords@1.5.1" has incorrect peer dependency "ajv@>=4.10.0".
```

Reply:
but people will waste time asking about it and trying to track it down, better to have something to point to

Reply:
When people see the problem, they either won't care or won't look into README page to find the answer, sooner they will search Google.

Regardless the fact I disagree with the principle I will add this as there is no harm and I want to make you happy ;)